data/naltok weapons.txt

```diff
@@ -44,7 +44,7 @@ outfit "Breaker Turret"
 	weapon
 		sprite "projectile/breaker"
 		"hardpoint sprite" "hardpoint/breaker"
-		"hardpoint offset"
+		"hardpoint offset" 5
```

Comment:
This would need to be changed again when the turret has a hardpoint sprite.

Reply:
Yeah, that fix was just to stop some warnings